### ISSUE TITLE
Current configuration

### ISSUE THREAD

**Issue by alexisLefebvre** [alexislefebvre/SymfonyWorkflowStyleBundle#3]
Current configuration for a transition :

```yaml
                    metadata:
                        title: transition submit title
                        style:
                            label: 'My custom label'
                            arrow_color: '#0088FF'
                            label_color: 'Red'
```

**Comment by alexisLefebvre** [comment]
Use `dump_style` instead of `style`?

**Comment by alexisLefebvre** [comment]
Fixed in #10
